src/main/java/com/smartparkinglot/backendservice/controllers/DriverController.java

```diff
@@ -0,0 +1,53 @@
+package com.smartparkinglot.backendservice.controllers;
+
+
+import com.smartparkinglot.backendservice.domain.Driver;
+import com.smartparkinglot.backendservice.services.driverservice.DriverService;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import org.apache.coyote.Response;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("api/user")
+@AllArgsConstructor
+public class DriverController {
+    private DriverService userService;
+    @GetMapping
+    public ResponseEntity<List<Driver>> getAllDrivers(){
+        return userService.getAllDrivers();
```

Comment:
Creating response entities at controller level is best practice. I mean we should separate concerns between controller and service. My suggestion is like this for all endpoints.
```java
return new ResponseEntity(userService.getAllDrivers(), HttpStatus.OK);
```
The difference is now service is just related to returning value from repository. We let the creating response at the controller level. And of course you need to update the DriverService too.